packages/driver/cypress/integration/e2e/multi-domain/commands/multi_domain_spies_stubs_clocks.spec.ts

```diff
@@ -25,6 +25,48 @@ context('multi-domain spies, stubs, and clock', { experimentalSessionSupport: tr
     })
   })
 
+  context('resets stubs', () => {
+    it('creates the stub', () => {
+      cy.switchToDomain('foobar.com', () => {
+        const stubEnv = cy.stub(Cypress, 'env').withArgs('foo').returns('bar')
```

Comment:
we might need to pay attention to any changes that happen in #20102 to make sure they don't impact the tests here. Currently, as long as there isn't a `foo` get in the `env` I think both PRs should work harmoniously.

Reply:
Since I'm stubbing `Cypress.env` I don't think it will matter but I wonder if your test should have an `after` block that resets the `config` and `env`?